packages/backend/src/ee/services/ai/agents/agentV2.ts

```diff
@@ -64,74 +64,6 @@ const getAgentTelemetryConfig = (
         },
     } as const);
 
-const getRepairToolCall =
```

Comment:
with this removal, you can have just an import like this on this file   
```

```
import { generateText, smoothStream, stepCountIs, streamText } from 'ai';
```

```